website/project/model.py

```diff
@@ -1776,11 +1777,11 @@ def get_recent_logs(self, n=10):
         """
         return list(reversed(self.logs)[:n])
 
-    @property
-    def date_modified(self):
-        '''The most recent datetime when this node was modified, based on
+    def date_updated(self):
```

Comment:
This method would be unused after migration, please remove it and add equivalent logic to the migration.